File: java/src/main/java/com/touzhijia/platform/entity/CreateUserReq.java
package com.touzhijia.platform.entity;

public class CreateUserReq {
	// 投之家用户名
	private String username;
	
	// 手机号码
	private String telephone;
	
	// 电子邮箱
	private String email;
	
	public class IdCard {
		// 身份证号
		private String number;
		// 实名
		private String name;
		
		public String getNumber() {
			return number;
		}
		public void setNumber(String number) {
			this.number = number;
		}
		public String getName() {
			return name;
		}
		public void setName(String name) {
			this.name = name;
		}
	}
	
	public class BankCard {
		private String number;
		private String bankName;
		private String branch;
		public String getNumber() {
			return number;
		}
		public void setNumber(String number) {
			this.number = number;
		}
		public String getBankName() {
			return bankName;
		}
		public void setBankName(String bankName) {
			this.bankName = bankName;
		}
		public String getBranch() {
			return branch;
		}
		public void setBranch(String branch) {
			this.branch = branch;
		}
	}
	
	// 身份证信息
	private IdCard idCard;
	
	// 银行卡信息
	private BankCard bankCard;

	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public String getTelephone() {
		return telephone;
	}

	public void setTelephone(String telephone) {
		this.telephone = telephone;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public IdCard getIdCard() {
		return idCard;
	}

	public void setIdCard(IdCard idCard) {
		this.idCard = idCard;
	}

	public BankCard getBankCard() {
		return bankCard;
	}

	public void setBankCard(BankCard bankCard) {
		this.bankCard = bankCard;
	}
}

File: csharp/touzhijia/entity/CreateUserReq.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace touzhijia.entity
{
    public class CreateUserReq
    {
        // 投之家用户名
        public String username{ get; set; }

        // 手机号码
        public String telephone{ get; set; }

        // 电子邮箱
        public String email{ get; set; }

        public class IdCard
        {
            // 身份证号
            public String number{ get; set; }
            // 实名
            public String name{ get; set; }

        }

        public class BankCard
        {
            public String number{ get; set; }
            public String bankName{ get; set; }
            public String branch{ get; set; }
        }

        // 身份证信息
        public IdCard idCard{ get; set; }

        // 银行卡信息
        public BankCard bankCard{ get; set; }
    }
}
